### PR TITLE
test: add cypress testing suite to chronograf

### PR DESCRIPTION
**Pull request by Iszy-Ami**
This PR adds new e2e cypress tests and configures Circle CI to execute them automatically on every git change.
Testing of components in Chronograf, especially for creating dashboard (variables), setting graf in explorer, manual adding data through write data button and snapshot testing of single stats in grafs.

  
  
  - [x] Rebased/mergeable
  - [x] Tests pass

### CODE REVIEW

File: etc/scripts/influxdb-onboarding.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```
Comment (rhajek) [inline etc/scripts/influxdb-onboarding.sh:1]: An onboarding script is not necessary now, it it possible to initialise database using environment properties like this:

```
docker run -d -p 8086:8086 \
      -v $PWD/data:/var/lib/influxdb2 \
      -v $PWD/config:/etc/influxdb2 \
      -e DOCKER_INFLUXDB_INIT_MODE=setup \
      -e DOCKER_INFLUXDB_INIT_USERNAME=my-user \
      -e DOCKER_INFLUXDB_INIT_PASSWORD=my-password \
      -e DOCKER_INFLUXDB_INIT_ORG=my-org \
      -e DOCKER_INFLUXDB_INIT_BUCKET=my-bucket \
      -e DOCKER_INFLUXDB_INIT_RETENTION=1w \
      -e DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=my-super-secret-auth-token \
      influxdb:2.0
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,3 +1,14 @@
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.1.3
+
+commands:
+  influxdb-onboarding:
+    steps:
+      - run:
+          name: "Post onBoarding request to InfluxDB 2"
```
Comment (rhajek) [inline .circleci/config.yml:9]: onboarding script s not necessary, you can use env properties to setup docker instance:
```
docker run -d -p 8086:8086 \
      -v $PWD/data:/var/lib/influxdb2 \
      -v $PWD/config:/etc/influxdb2 \
      -e DOCKER_INFLUXDB_INIT_MODE=setup \
      -e DOCKER_INFLUXDB_INIT_USERNAME=my-user \
      -e DOCKER_INFLUXDB_INIT_PASSWORD=my-password \
      -e DOCKER_INFLUXDB_INIT_ORG=my-org \
      -e DOCKER_INFLUXDB_INIT_BUCKET=my-bucket \
      -e DOCKER_INFLUXDB_INIT_RETENTION=1w \
      -e DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=my-super-secret-auth-token \
      influxdb:2.0
```

**Review by rhajek — CHANGES_REQUESTED** [review]
Please remove unnecessary influxdb-onboarding.sh script.

**Comment by sranka** [comment]
Closing, feel free to create a new PR after applying a new strategy for testing (InfluxDB enterprise first).

**Pull request closed without merging**